clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/model/AzureNamedImage.groovy

```diff
@@ -0,0 +1,17 @@
+package com.netflix.spinnaker.clouddriver.azure.resources.vmimage.model
+
+/**
+ * Created by larry on 3/10/16.
+ */
+class AzureNamedImage {
+  String imageName
+  Boolean iscustom = false
```

Comment:
I know you're just moving this class here, but it reads a bit funny to have `iscustom` as opposed to `isCustom`.